NEWS.txt

```diff
@@ -1,3 +1,14 @@
+Version 1.4.2 - 2017-05-08
+** Bug
+    * [PICARD-969] - Search dialog webservices get queued behind matched album requests
+    * [PICARD-1034] - Picard not seeing TOPE and TOAL
+    * [PICARD-1053] - Picard does not stop analyzer while moving
+    * [PICARD-1055] - picard hangs with: RuntimeError: maximum recursion depth exceeded in cmp
+    * [PICARD-1077] - ID3v2.4 text encoding settings are not saved correctly
+    
+** Improvement
+    * [PICARD-1070] - The "Convert Unicode punctuation characters to ASCII" function only works in certain tags
```

Comment:
These are all directly generated from jira. You can update the issues on jira. They were all filed/fixed by you iirc.